examples/systemcalls/systemcalls.c

```diff
@@ -80,20 +159,82 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
         command[i] = va_arg(args, char *);
     }
     command[count] = NULL;
-    // this line is to avoid a compile warning before your implementation is complete
-    // and may be removed
-    command[count] = command[count];
 
+    bool b_status = false;
+    const pid_t pid = fork();
```

Comment:
@krish0706 think about how you could avoid duplicating logic with the function above.